scripts/quick_install.sh

```diff
@@ -170,7 +170,7 @@ helm upgrade --install merlin caraml/merlin --namespace=mlp --values=./values-e2
   --version ${MERLIN_CHART_VERSION} \
   --set deployment.image.tag=${VERSION} \
   --set ui.oauthClientID=${OAUTH_CLIENT_ID} \
-  --set mlpApi.apiHost=http://mlp.mlp.${INGRESS_HOST}.nip.io/v1 \
+  --set mlpApi.apiHost=http://mlp.mlp.${INGRESS_HOST}.nip.io \
```

Comment:
Thanks for updating the host to drop the `/v1`.

(Guess we can update the global / default values for the charts, once this change is made to turing / xp as well. Will make a note of this.)

Reply:
I am updating chart for both merlin and turing as well.